modules/aws-emr-config/versions.tf

```diff
@@ -1,9 +1,3 @@
 terraform {
   required_version = ">= 0.13"
-  required_providers {
-    aws = {
```

Comment:
I'm not sure this was duplicate, since you can use the nested modules directly and wouldn't have the provider requirements from the root module.

In fact, the root module doesn't technically need the aws provider, although it is nice to specify because it'll generate the requirements tables in the README.

Make sure to run `make terraform/docs` to get the updated README.

Reply:
@souza-dan it looks like the makefile does not authenticate with Github SSO: `ERROR: The 'Datatamer' organization has enabled or enforced SAML SSO.
To access this repository, you must use the HTTPS remote with a personal access token or SSH with an SSH key and passphrase that has been authorized for this organization.`

Is there a way I should be authenticating?

Reply:
I recommend setting up your ssh key in github. I think these are the two docs to follow to get that working:
https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account
https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-an-ssh-key-for-use-with-saml-single-sign-on

Then check out the repo again:
`git clone git@github.com:Datatamer/terraform-aws-emr.git`

and the `make terraform/docs` should work. I just tried it out, so if you're stuck let me know and I can push a commit with it applied.